docs/user-guide/annotations.md

```diff
@@ -500,3 +503,14 @@ nginx.ingress.kubernetes.io/lua-resty-waf-extra-rules: '[=[ { "access": [ { "act
 ```
 
 For details on how to write WAF rules, please refer to https://github.com/p0pr0ck5/lua-resty-waf.
+
+### Proxy protocol
+
+Enable or disable proxy proxy protocol for SSL PassThrough backend.
```

Comment:
"proxy proxy protocol" -> "Proxy Protocol"
"PassThrough" -> "Passthrough"
"for **a**"